backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -162,6 +164,11 @@ protected void setActionMessageParameters() {
     }
 
     protected boolean validateCreateImage() {
+        StorageDomain storageDomain = storageDomainDao.get(getParameters().getStorageDomainId());
+        if (!isSupportedByManagedBlockStorageDomain(storageDomain)) {
+            return failValidation(EngineMessage.ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_FOR_MANAGED_BLOCK_STORAGE_TYPE);
```

Comment:
I think I commented on that part within the other PR but this is the right place - shouldn't ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_FOR_MANAGED_BLOCK_STORAGE_TYPE be added to the errors by the call to `isSupportedByManagedBlockStorageDomain` above?